src/test/java/utilities/Util.java

```diff
@@ -1,7 +1,39 @@
 package utilities;
 
+import java.util.List;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.Select;
+
 public class Util {
 
+	/*
+	 * 
+	 * select multi option
+	 * @Author: Sedef Akyo
+	 */
+	
+
+	public static void selectMulti(WebDriver driver, String xpathLocator, String optionValue) {
```

Comment:
1, To make this function more reusable, pass in a `By` object instead of xpath
2, The `optionValue` could be an array of string, then you can select everything in the array
3, Fix formatting